src/test/services/user.test.ts

```diff
@@ -0,0 +1,246 @@
+import { USER_ROLES, UserRole } from "../../IsaacApiTypes";
+import {
+  extractTeacherName,
+  isAdmin,
+  isAdminOrEventManager,
+  isEventLeader,
+  isEventLeaderOrStaff,
+  isEventManager,
+  isLoggedIn,
+  isStaff,
+  isStudent,
+  isTeacher,
+  isTeacherOrAbove,
+  isTeacherPending,
+  isTutor,
+  isTutorOrAbove,
+  roleRequirements,
+  schoolNameWithPostcode,
+} from "../../app/services/user";
```

Comment:
Isn't this the same as `"../../app/services"`?

Reply:
Yes, looks like the auto import just decided to specify `/user` - both work, but I've removed the /user now